content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,507 @@
+it has an initial opinion:
+
+     opinion
+        <-- choose local truth computation of {YES, NO, UNDECIDED}
+    
+The algorithm proceeds in rounds for each node.  The liveness of the
+algorithm is severely constrained in the absence of timeouts for a
+round to proceed.
+
+### Setup Parameters
+
+The node initializes the following integer constants:
+
+First the constants:
+
+    ;;; The following values are constants chosen with justification from experiments
+    ;;; performed with the adversarial models
+
+    ;; FIXME: find and justify an empirical confidence threshold 
+    confidence_threshold
+     <-- T  ;; BOGUS:  this should be a function of the network size
+            ;; and the current confidence in the observer majority
+    
+    ;; constant look ahead parameter
+    look_ahead 
+      <-- 19
+    
+    ;; first order confidence smoothing parameter (aka $alpha_1$)
+    certainty
+      <-- 4/5  ;; empirically justified from consensus simulation
+    
+    ;; second order confidence smoothing parameter (aka $alpha_2$)
+    doubt
+      <-- 2/5  ;; empirically justified from consensus simulation
+
+    ;; neighbor threshold multiplier
+    k_multiplier 
+      <-- 2
+    ;; maximal threshold multiplier, i.e. we will never exceed 
+    ;; questioning k_multiplier * max_k_multiplier peers
+    max_k_multiplier 
+      <-- 4
+      
+The following variables will keep the state of Glacier:
+      
+    ;; total number of votes examined over all rounds
+    total_votes 
+      <-- 0 
+    ;; total number of YES (i.e. positive) votes for the truth of the proposal
+    total_positive 
+      <-- 0
+    ;; current number of nodes to attempt to query in a round
+    k 
+      <-- 7
+
+
+###  Query 
+
+A node selects $k$ nodes randomly from the current locally known
+complete set of peers in the network.  
+
+A query is sent to each neighbor with the node's current `opinion` of
+the proposal.
+
+Each node replies with their current opinion on the proposal.
+
+When the query finishes, the node now initializes the following two
+values:
+
+    new_votes 
+      <-- |total vote replies received in this round to the current query|
+    positive_votes 
+      <-- |YES votes received from the query| 
+    
+### Computation    
+
+The node runs the `new_votes` and `positive_votes` parameters received
+in the query round through the following algorithm:
+
+    total_votes 
+      += new_votes
+    total_positive 
+      += positive_votes
+    confidence 
+      <-- total_votes / (total_votes + look_ahead) 
+    total_evidence 
+      <-- total_positive / total_votes
+    new_evidence 
+      <-- positive_votes / new_votes
+    evidence 
+      <-- new_evidence * ( 1 - confidence ) + total_evidence * confidence 
+    alpha 
+      <-- doubt * ( 1 - confidence ) + certainty * confidence 
+    
+### Opinion 
+
+The node updates its local opinion on the consensus proposal:
+
+    IF
+      $evidence$ > $alpha$ 
```

Comment:
Remove LaTex prefixes here.  Include only in appendices until we debug the presentation run.